aea/helpers/exec_timeout.py

```diff
@@ -227,6 +229,7 @@ def stop(cls, force: bool = False) -> None:
                 if cls._supervisor_thread and cls._supervisor_thread.is_alive():
                     cls._supervisor_thread.join()
                 cls._supervisor_thread = None
+                cls._start_count = 0
```

Comment:
what's that doing?

Reply:
it's a counter of number it was started, kind of usage references, and here is set to 0 that means it's not in use. this class can be shared across several agents but should have only one thread started